docs/content/what-are-the-differences-between-temporal-and-a-traditional-system.md

```diff
@@ -0,0 +1,64 @@
+---
+id: what-are-the-differences-between-temporal-and-a-traditional-system
+title: What are the differences between Temporal and a traditional system?
+description: This page compares a function execution in a traditional system and a Workflow Execution in Temporal.
+tags:
+  - explanation
+---
+
+import DetermineHeader from '../components/DetermineHeader.js'
+
+export const headingText = 'What are the differences between Temporal and a traditional system?'
+
+<DetermineHeader
+hLevel={props.heading}
+hText={headingText}
+/>
+
+The Temporal platform exists to facilitate Workflow Executions.
+This page compares a function execution in a traditional system and a Workflow Execution in Temporal.
+
+![Diagram that shows traditional function execution and Temporal Workflow Execution](/img/docs/function-execution-vs-reentrant-process.png)
+
+:::note Legend
+
+λ = traditional function execution  
+π = Temporal Workflow Execution  
+𝝈 = state
+
+:::
+
+Although the two systems seem similar at first glance, they differ in several significant ways.
+
+## Failure
```

Comment:
One thing I realized is that we will be saving ourselves some trouble if we try to avoid any headings in the content pages.
Maybe make these sections just **bold text** ?
The page is short enough, I don't see a need for someone to need to link to one of these directly.

Reply:
For now, let's go with bold. However, we need to circle back on this issue because headings are important for both accessibility and SEO.